moto/cognitoidp/utils.py

```diff
@@ -7,6 +7,34 @@
 import base64
 
 
+PAGINATION_MODEL = {
```

Comment:
This model was used in SFN first, but can be reused for every service like this:
```
method:
    input_token: name of the method-kwarg that contains the token
    limit_key: name of the method-kwarg that contains the limit/max/nr of results the user asked for
    page_ending_range_keys: attributes used to create unique hash for the item, required to identify the starting point for the next token